<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>626</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Calculdora</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>calculator.ico</normaloff>calculator.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#000;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>381</width>
      <height>621</height>
     </rect>
    </property>
    <widget class="QLabel" name="resultado">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>10</y>
       <width>361</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#fff;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>540</y>
       <width>171</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #606060;
border-radius:35px;
color: #fff;
font-size: 25px;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMaisMenos">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ccc;
border-radius:35px;
color: #000;
font-size: 25px;</string>
     </property>
     <property name="text">
      <string>+/-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>440</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #606060;
border-radius:35px;
color: #fff;
font-size: 25px;</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn9">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>240</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #606060;
border-radius:35px;
color: #fff;
font-size: 25px;</string>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMenos">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>340</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: orange;
border-radius:35px;
color: #fff;
font-size: 25px;</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #606060;
border-radius:35px;
color: #fff;
font-size: 25px;</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>340</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #606060;
border-radius:35px;
color: #fff;
font-size: 25px;</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>340</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #606060;
border-radius:35px;
color: #fff;
font-size: 25px;</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn8">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>240</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #606060;
border-radius:35px;
color: #fff;
font-size: 25px;</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMais">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>440</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: orange;
border-radius:35px;
color: #fff;
font-size: 25px;</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #606060;
border-radius:35px;
color: #fff;
font-size: 25px;</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnVirgula">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>540</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #606060;
border-radius:35px;
color: #fff;
font-size: 25px;</string>
     </property>
     <property name="text">
      <string>,</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ccc;
border-radius:35px;
color: #000;
font-size: 25px;</string>
     </property>
     <property name="text">
      <string>AC</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMult">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>240</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: orange;
border-radius:35px;
color: #fff;
font-size: 25px;</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDivisao">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>140</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: orange;
border-radius:35px;
color: #fff;
font-size: 25px;</string>
     </property>
     <property name="text">
      <string>รท</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>440</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #606060;
border-radius:35px;
color: #fff;
font-size: 25px;</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnIgual">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>540</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: orange;
border-radius:35px;
color: #fff;
font-size: 25px;</string>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPorcentagem">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ccc;
border-radius:35px;
color: #000;
font-size: 25px;</string>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #606060;
border-radius:35px;
color: #fff;
font-size: 25px;</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
